### PR TITLE
Added IDEA formatting for code style.

### PR DESCRIPTION
**Pull request by freeone3000**

### CODE REVIEW

File: idea_maluuba_formatter.xml [hunk 1]
```diff
@@ -0,0 +1,86 @@
+<code_scheme name="MaluubaStyle">
+  <option name="OTHER_INDENT_OPTIONS">
+    <value>
+      <option name="INDENT_SIZE" value="2" />
```
Comment (juharris) [inline idea_maluuba_formatter.xml]: Don't we normally use 4?

File: idea_maluuba_formatter.xml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      <option name="CONTINUATION_INDENT_SIZE" value="4" />
+      <option name="TAB_SIZE" value="2" />
```
Comment (juharris) [inline idea_maluuba_formatter.xml]: Use 4

File: idea_maluuba_formatter.xml (hunk 1 above)
Comment (freeone3000) [inline idea_maluuba_formatter.xml]: I find it most impressive that you actually reviewed my XML.

File: idea_maluuba_formatter.xml [hunk 3]
```diff
@@ -0,0 +1,86 @@
+<code_scheme name="MaluubaStyle">
+  <option name="OTHER_INDENT_OPTIONS">
+    <value>
+      <option name="INDENT_SIZE" value="4" />
```
Comment (juharris) [inline idea_maluuba_formatter.xml]: Does this file match it's own settings?

Comment (freeone3000) [inline idea_maluuba_formatter.xml]: ...Yes?

Comment (freeone3000) [inline idea_maluuba_formatter.xml]: No, because the file is XML, and the code settings are for Java.

**Pull request merged**